addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -380,6 +380,8 @@ export class MassMailingHtmlField extends HtmlField {
                     selection.removeAllRanges();
                     selection.addRange(range);
                 }
+                // mark selection done for tour testing
+                $editable.addClass('theme_selection_done');
```

Comment:
Add a class here...

There doesn't seem to be a reliable way to know this one is finished without plugging into the JS, or somehow waiting for the events to be fully processed after the click.